doc/man/s3fs.1

```diff
@@ -326,6 +326,10 @@ Otherwise an error is returned.
 \fB\-o\fR requester_pays (default is disable)
 This option instructs s3fs to enable requests involving Requester Pays buckets (It includes the 'x-amz-request-payer=requester' entry in the request header).
 .TP
+\fB\-o\fR mime (default is "/etc/mime.types")
+Specify the path of the mime.types file.
+If not specified, use the "/etc/mime.types" file ("/etc/apache2/mime.types" for macOS) as the default value.
```

Comment:
Are you sure you want `/etc/apache2/mime.types` for macOS in all cases?

Reply:
I would like to discuss this point. That's because I'm still worried and still can't judge.
Details are described in the comments below.

Reply:
Same as COMPILATION.md.